src/index.js

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './components/App.js.jsx';
-import './css/index.css';
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { Router, Route, browserHistory } from 'react-router'
+import App from './components/App.js.jsx'
+import BuilderContainer from './components/containers/BuilderContainer.js.jsx'
+import DashboardContainer from './components/containers/DashboardContainer.js.jsx'
+import NewDay from './components/UI/NewDay.js.jsx'
+import SavedDays from './components/UI/SavedDays.js.jsx'
+import './css/index.css'
 
 ReactDOM.render(
-  <App />,
+  <Router history={ browserHistory }>
+    <Route path="/" component={ App }>
+      <Route path="/my-account" component={ DashboardContainer }>
```

Comment:
Having no experience in routing previously, it seems like routing in React is pretty straight forward - maybe that's a testament to how clean your code is.  I also think you did with the naming of your routes with RESTful architecture.  I also agree with Andrew on keeping the user-account named :my-account.  Just throw a validation in to make sure that the user can only see the account for their ID and you're good to go.